docs/stories/2.6.openai-custom-endpoint.md

```
@@ -0,0 +1,161 @@
+- [x] Refresh documentation and operator runbooks (AC: 1, 4).
+  - [x] Add `OPENAI_BASE_URL` guidance to `.env.example` and architecture overviews.
+  - [x] Update troubleshooting/runbook sections covering OpenAI configuration and telemetry expectations.
+  - [x] Capture QA/PO awareness of the new knob within relevant assessment artifacts.
+
+## Testing
+- [x] `PYTHONPATH=src pytest tests/unit/config/test_openai_settings.py tests/unit/cli/test_openai_client.py`
+- [x] `PYTHONPATH=src pytest tests/unit/scripts/test_ask_qdrant.py::test_embedding_cli_base_url`
+- [x] `PYTHONPATH=src pytest tests/unit/fancyrag/kg/test_pipeline.py::test_shared_client_base_url`
+
+## Change Log
+| Date       | Version | Description                                   | Author |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)